samples/notebooks/Z-Tests/run-user-regression-notebooks.ipynb

```diff
@@ -162,7 +162,8 @@
     "                    }],\n",
     "            \"compute\": {\n",
     "              \"container\" : {\n",
-    "                  \"p_concurrent\": \"1\"\n",
+    "                  \"p_concurrent\": \"1\",\n",
+    "                  \"profile\": \"small\"\n",
```

Comment:
is this a json object? if so the hanging , at the end of this line is invalid

Reply:
that is not actually in the json.....that is github doing a literal string translation...(the \n for carriage return, etc)